batavia/builtins/hash.js

```diff
@@ -1,8 +1,8 @@
 var exceptions = require('../core').exceptions
 var type_name = require('../core').type_name
-var types = require('../types')
 
 function hash(args, kwargs) {
+    var types = require('../types')
```

Comment:
It's a fair point that something must be done.
However require on each call to `hash()` seems like a crazy penalty, doesn't it?

I'm assuming it's going to be used a lot.
P.S. I don't see `hash` being used by `types` anyway...